mutiny-wasm/src/indexed_db.rs

```diff
@@ -212,6 +232,18 @@ fn used_once(key: &str) -> bool {
     )
 }
 
+/// To help prevent force closes we save to local storage as well as indexed db.
+/// This is because indexed db is not always reliable.
+///
+/// We need to do this for the channel manager and channel monitors.
+fn write_to_local_storage(key: &str) -> bool {
+    match key {
+        CHANNEL_MANAGER_KEY => true,
```

Comment:
this should also be starts_with, because it looks like: `manager_c324484a-6de1-48fb-91c8-cf506db6ac79`

Reply:
good catch